taskcluster/docker/base/Dockerfile

```diff
@@ -28,17 +28,28 @@ RUN apt-get update -qq \
                           python3-yaml \
                           locales \
                           git \
+                          git-lfs \
                           tmux \
                           htop \
                           vim \
                           nano \
                           ncurses-term \
+                          openssh-client \
                           zstd \
                           screen \
                           curl \
     # Clean up the apt cache to reduce image size.
     && apt-get clean
 
+# Add GitHub's SSH host key to known_hosts file for the root user
+RUN mkdir -p /root/.ssh && \
+    ssh-keyscan github.com >> /root/.ssh/known_hosts
+
+# Add GitHub's SSH host key to known_hosts file for the worker user
+RUN mkdir -p /builds/worker/.ssh && \
+    ssh-keyscan github.com >> /builds/worker/.ssh/known_hosts && \
```

Comment:
It would be better to check in the known good keys directly rather than scan for them with each build, to avoid MitM possibilities with each rebuild. https://searchfox.org/mozilla-central/source/taskcluster/docker/partner-repack has an example of what we do elsewhere.

Reply:
Thank you @bhearsum! I'll get to modifying the behavior tomorrow!